analyzer/src/funTest/assets/projects/synthetic/poetry/pyproject.toml

```diff
@@ -0,0 +1,80 @@
+[tool.poetry]
+name = "example"
+version = "1.0.0"
+description = "example project"
+authors = ["author name <author@author.com>"]
+packages = [
+    {include = "example"}
+]
+include = [
+    {path = "example/py.typed"},
+]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+pydantic = "^1.9.1"
+boto3 = "^1.24.0"
+dateparser = "^1.1.1"
+jmespath = "^1.0.0"
+packaging = "^21.3"
+
+[tool.poetry.dev-dependencies]
+pytest = "^7.1.2"
+pytest-cov = "^3.0.0"
+mypy = "^0.960"
+xenon = "^0.9.0"
+radon = "^5.1.0"
+pylint = "^2.13.9"
+jsonschema = "^4.5.1"
+types-dateparser = "^1.1.2"
+yapf = "^0.32.0"
+isort = "^5.10.1"
+boto3-stubs = {extras = ["s3"], version = "^1.24.0"}
+pylint-pydantic = "^0.1.4"
+coverage = "^6.4"
+safety = "^1.10.3"
+types-jmespath = "^0.10.2"
```

Comment:
It would be good to reduce the amount of dependencies in the test project. This would reduce the size of the expected result file and speed up the test.